contracts/src/v0.8/automation/dev/v2_2/AutomationRegistryBase2_2.sol

```diff
@@ -792,12 +797,13 @@ abstract contract AutomationRegistryBase2_2 is ConfirmedOwner, ExecutionPreventi
   function _validateLogTrigger(
     uint256 upkeepId,
     bytes memory rawTrigger,
-    UpkeepTransmitInfo memory transmitInfo
+    HotVars memory hotVars
   ) internal returns (bool, bytes32) {
     LogTrigger memory trigger = abi.decode(rawTrigger, (LogTrigger));
     bytes32 dedupID = keccak256(abi.encodePacked(upkeepId, trigger.logBlockHash, trigger.txHash, trigger.logIndex));
     if (
-      (trigger.blockHash != bytes32("") && _blockHash(trigger.blockNum) != trigger.blockHash) ||
+      (!hotVars.skipReorgProtection &&
```

Comment:
nitpick - this is a double negative. Could we change the variable name to `reorgProtectionEnabled`? or similar? I think it would read better. Then you could do... `if (hotVars.reorgProtectionEnabled && ...)`

Reply:
done